src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -0,0 +1,61 @@
+package ru.hogwarts.school.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ru.hogwarts.school.model.Student;
+import ru.hogwarts.school.service.StudentService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/student")
+public class StudentController {
+    private StudentService studentService;
+
+    public StudentController(StudentService studentService) {
+        this.studentService = studentService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Student> createStudent(@RequestBody Student student) {
+        return ResponseEntity.ok(student);
+    }
```

Comment:
Не хватает вызова метода сохранения
```suggestion
    @PostMapping
    public ResponseEntity<Student> createStudent(@RequestBody Student student) {
        return ResponseEntity.ok(studentService.createStudent(student));
    }
```